BaristaSwift/Podfile

```diff
@@ -11,7 +11,6 @@ def barista_pods
   pod 'SmartStore', :path => 'mobile_sdk/SalesforceMobileSDK-iOS'
   pod 'SmartSync', :path => 'mobile_sdk/SalesforceMobileSDK-iOS'
   pod 'SalesforceSwiftSDK', :path => 'mobile_sdk/SalesforceMobileSDK-iOS'
-  pod 'PromiseKit', :git => 'https://github.com/mxcl/PromiseKit', :tag => '5.0.3'
```

Comment:
We have upgraded `PromiseKit` and don't need the explicit declaration anymore. It is pulled through `SalesforceSwiftSDK`.